rules/falco_rules.yaml

```diff
@@ -3083,6 +3083,30 @@
   priority: CRITICAL
   tags: [syscall, mitre_defense_evasion]
 
+- list: ingress_remote_file_copy_binaries
+  items: [wget, curl]
```

Comment:
`curl` scenario might be a bit tricky as it could be used as sending data out. What I would suggest is that make an macro for curl with the following condition:

```
macro: curl_download
condition: proc.name = curl and (proc.cmdline contains (" > ") or proc.cmdline contains (" | "))
```

Reply:
@Kaizhe done. Please take another look.